src/util/Date.cpp

```diff
@@ -281,3 +281,12 @@ std::optional<int> DateOrLargeYear::getDay() const {
     return 1;
   }
 }
+
+// _____________________________________________________________________-
+std::optional<std::string> DateOrLargeYear::getStrTimezone() const {
+  if (isDate()) {
+    return getDateUnchecked().formatTimeZone();
+  } else {
+    return std::nullopt;
```

Comment:
I am not sure if we shouldd return nullopt or the empty string here.
I think you could also argue that the years only have to timeZone, so you can just return the empty string here.
I however have no strong optinion. 
In any case this case should be tested.